templates/partial/term-index.html.twig

```diff
@@ -1,6 +1,6 @@
 <!DOCTYPE html>
```

Comment:
Is the doctype really meant to be here in a partial?

Reply:
Ah good catch! Removed now.